public-api/src/main/resources/public-api.yaml

```diff
@@ -119,7 +120,7 @@ paths:
             $ref: "#/definitions/DbDomainListResponse"
 
   /v1/databrowser/advanced-concept-search:
```

Comment:
Let's make sure this is the same api name/path as the private api

Reply:
So the workbench API actually has the workspace ID in the path (since we will eventually be using workspace metadata when querying for stuff). We could name it similarly, but the paths are going to differ.

Reply:
cool. I like things named the same that are the same.

Reply:
Changed the name to searchConcepts now that the old concept-search is removed.